baus/datasources.py

```diff
@@ -196,8 +198,15 @@ def final_year():
 
 
 @orca.injectable(cache=True)
-def store(paths):
-    return pd.HDFStore(os.path.join(orca.get_injectable("inputs_dir"), paths["store"]))
+def store(run_name):
+    # retrieve h5 from inputs folder
+    h5 = os.path.join(orca.get_injectable("inputs_dir"), "basis_inputs/parcels_buildings_agents/2015_09_01_bayarea_v3.h5")
+    # copy it to outputs folder for reading and writing during run time
+    coresum_output_dir = pathlib.Path(orca.get_injectable("outputs_dir")) / "core_summaries"
+    coresum_output_dir.mkdir(parents=True, exist_ok=True)
+    shutil.copyfile(h5, os.path.join(coresum_output_dir, "{}_h5_store.csv").format(run_name))
```

Comment:
Two questions, @theocharides : 
1. If the upshot of the conversation [here](https://app.asana.com/0/385259290425521/1206162709600867/f) is that we can safely set a read only flag, multiple users can point to the same store concurrently, and we don't need to copy the file I assume?
1. How come the h5 file  gets copied to a `"{}_h5_store.csv"` target?

Reply:
That convo was saying that the timestamp for this particular h5 is updated at the end of each run.